collector/pg_database.go

```diff
@@ -79,32 +80,39 @@ func (c PGDatabaseCollector) Update(ctx context.Context, instance *instance, ch
 	var databases []string
 
 	for rows.Next() {
-		var datname string
+		var datname sql.NullString
```

Comment:
Is this column nullable? The documentation doesn't mention that this field is nullable.